test/zdtm/static/sched_policy00.c

```diff
@@ -51,7 +51,7 @@ int main(int argc, char **argv)
 	}
 
 	p.sched_priority = param;
-	if (sched_setscheduler(pid, SCHED_RR, &p)) {
+	if (sched_setscheduler(pid, SCHED_RR | SCHED_RESET_ON_FORK, &p)) {
```

Comment:
Usually, we don't mix test and criu changes in the same commit...

Reply:
Thank you Andrei for the code review! I have updated the pull request.